### PR TITLE
Add msedge to sauce config.

### PR DESCRIPTION
**Pull request by jdalton**
Add msedge to sauce config using the short name for the platform (will see if that works).

### CODE REVIEW

File: test/browser/sauce-browsers.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 [
   {
+    "browserName": "microsoftedge",
+    "platform": "WIN10",
+    "version": "20.10240"
```
Comment (patrickkettner) [inline test/browser/sauce-browsers.json]: can we leave off `version`? normally sauce will default to `latest` when this is done, no?"

Comment (jdalton) [inline test/browser/sauce-browsers.json]: Maybe, though why would we be any different than Chrome/Firefox?

I've got  a thread going with them to change the version from `20.x` to `12.x` based on our [blog guidance](http://blogs.windows.com/msedgedev/2015/09/21/understanding-versions-in-an-evergreen-browser/).

Comment (patrickkettner) [inline test/browser/sauce-browsers.json]: on our config?

it is behind, but it is supposed to represent the stable, beta, dev, and canary versions. if sauce only supports stable, then the default version would be ideal.

Comment (jdalton) [inline test/browser/sauce-browsers.json]: I think experimenting with dropping version numbers would be more approp for another pr.
